setup_support.py

```diff
@@ -56,6 +56,14 @@ def finalize_options(self):
         self.announce('Writing man page(s) to %s' % self.manpath)
         self._today = datetime.date.today()
 
+    def version(self):
```

Comment:
right -- let me repeat that there is no need for this one to be an instance method -- it doesn't use/need self.  Could be a static method, which would have resolved at least
```shell

 $> pip install -e .                              
Obtaining file:///home/yoh/proj/datalad/datalad
  Running setup.py (path:/home/yoh/proj/datalad/datalad/setup.py) egg_info for package from file:///home/yoh/proj/datalad/datalad
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/home/yoh/proj/datalad/datalad/setup.py", line 21, in <module>
        version = BuildManPage.version()
    TypeError: unbound method version() must be called with BuildManPage instance as first argument (got nothing instead)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/home/yoh/proj/datalad/datalad/setup.py", line 21, in <module>

    version = BuildManPage.version()

TypeError: unbound method version() must be called with BuildManPage instance as first argument (got nothing instead)
```
but imho, once again, better just to have it as a function since per se it has nothing to do with man page generator.  So move it outside of the class, and also verify that it all works ;)